tutornotes/templates/notes/build/notes/Dockerfile

```diff
@@ -1,20 +1,21 @@
 # syntax=docker/dockerfile:1.4
-FROM docker.io/ubuntu:20.04
+FROM docker.io/python:3.12-slim-bookworm
```

Comment:
Any reason for choosing this explicit image?

Reply:
@DawoudSheraz I checked this [comment](https://github.com/overhangio/tutor-discovery/pull/67/files#r1533458049) and I thought it was a good opportunity to test with a Python image to avoid using Pyenv. I picked the slim one to get an image as light as possible.

Reply:
Alright. If it builds and works as expected, that is good.